sections/semantics-embedded-content.include

```diff
@@ -3623,27 +3623,58 @@
   For images that have captions, markup generators should use the
   <{figure}> and <{figcaption}> elements to provide the image's caption.
 
-  As a last resort, implementors should either set the <code>alt</code> attribute to the empty string, under
-  the assumption that the image is a purely decorative image that
-  doesn't add any information but is still specific to the surrounding
-  content, or omit the <code>alt</code> attribute
-  altogether, under the assumption that the image is a key part of the
-  content.
+  As a last resort, implementors
+  <a href="https://www.w3.org/TR/ATAG20/#gl_b23">must omit the alt attribute altogether</a>.
+  [[!ATAG20]]
+
+  <div class="note">
+    Setting an <{image}> element's <{img/alt}> attribute to the empty string means the image in
+    question provides no essential information. Assistive technology, such as screen readers,
+    will completely ignore the presence of such an image, as ignoring it will not stop the user
```

Comment:
s/completely/typically/ ??

As far as I know that is the normal and rational behaviour, but I don't think it is the only possible behaviour. Allowing for other cases can reduce confusion when someone who has a strange setup - e.g. because they are not a habitual screen reader user - finds that something different occurs.

YMMV though - this is a question/suggestion rather than a formal request for change